R/pkg/inst/tests/testthat/test_utils.R

```diff
@@ -17,6 +17,9 @@
 
 context("functions in utils.R")
 
+# Ensure Spark is installed
+sparkCheckInstall()
```

Comment:
hmm, we could put it in https://github.com/apache/spark/blob/master/R/pkg/tests/run-all.R?